lib/modules/dosomething/dosomething_helpers/dosomething_helpers.share.inc

```diff
@@ -164,3 +164,53 @@ function dosomething_helpers_share_bar($path, $share_types, $share_description,
 
   return $share_bar . "</ul>";
 }
+
+/**
+ * Return social share types and custom content.
+ *
+ * @param object $campaign
+ *   The campaign object to get custom content from.
+ * @return array $social share types
+ *   An array of social share types and custom content.
+ */
+
+function dosomething_helpers_social_share_types($campaign) {
```

Comment:
Sorry this is super nitpicky. But i feel like this should be a helper function in the `dosomething_campaign.theme.inc` because it is specific to campaign page functionality, it's just being used in two places on the page. So then you should be able to call the helper once and then have that array set to use where ever you need it.